README.md

```diff
@@ -43,6 +43,12 @@ By default API requests are made over HTTP. HTTPS can be used by setting
 Pusher.encrypted = true
 ```
 
+If you created your app in a cluster different from the default one, you must pass the cluster option:
+
+``` ruby
+Pusher.cluster = 'your-app-cluster'
```

Comment:
We want to deprecate the singleton approach. If the user wants to use the cluster option he should create a new instance of the Pusher::Client instead.